lib/y2lan_restart_common.pm

```diff
@@ -149,14 +149,24 @@ sub check_network_status {
     }
     assert_script_run 'cat journal.log';                              # print journal.log
     my $is_empty = script_run('[ -s journal.log ]');
+    # empty journal.log indicating that service was not restarted or reloaded
     if ($is_empty) {
-        assert_equals($expected_status, 'no_restart', "Network restart didn't occur, even though it was expected.");
+        assert_equals($expected_status, 'no_restart', "Network service should have been restarted/reloaded. journalctl cannot find events");
     }
+    # Based on the query's filter it is assumed that the service is restarted otherwise it is reloaded.
     else {
-        assert_equals($expected_status, 'restart', "Network restart occurred, even though it wasn't expected.");
+        my $journal_findings = script_output('cat journal.log');
+        my $query_pattern    = "shutting down|ifdown all|Stopping wicked";
+        my @matched          = grep { $_ =~ /($query_pattern)/ } $journal_findings;
+        if (@matched) {
+            assert_equals($expected_status, "restart", "Network restart occurred, even though it wasn't expected.. journalctl found \n$journal_findings\n\n");
+        }
+        else {
+            assert_equals($expected_status, "reload", "Network reload occurred, even though it wasn't expected. Journalctl found \n$journal_findings\n\n");
```

Comment:
s/Journalctl/journalctl Or just make it consistent in both error messages.

Reply:
fixed